### PR TITLE
Continuation URL parameter for Mendix SSO

### PR DESCRIPTION
**Pull request by MarkvanMents**
For release 4.2.0 of Mendix SSO

### CODE REVIEW

File: content/en/docs/appstore/modules/mendix-sso.md [hunk 1]
```diff
@@ -274,6 +274,45 @@ Remember that data which comes from the end-user's Mendix ID via SSO (for exampl
...
+If your Mendix application has registered any custom domains, or if you want to redirect the end-user to a different application, the host names need to be specifically supplemented for your deployed application. This can be configured in the **Runtime** settings tab of the Developer Portal as described below.
+
+#### 5.4.2 Supplementing Allowed Continuation URLs
+
+On the **Runtime** tab of your app environment's [Environment Details](/developerportal/deploy/environments-details/#runtime-tab) you can set Custom Environment Variables. Here you can specify a comma-separated list of URLs that can be used as continuation URLs in your Mendix application.
+
+To add a list of allowed continuation URLs, do the following:
+
+1. Click **Add** for **Custom Environment Variables**.
+1. Select **Unsupported**.
```
Comment (MarkvanMents) [inline content/en/docs/appstore/modules/mendix-sso.md]: @zaipie 
If you add an unsupported environment variable, you get this message.
> Add an unsupported environment variable at your own risk. > Your app or its environment might crash or work incorrectly if wrongly configured.

I think we should be aiming to make this a supported environment variable if this is not a beta feature.

Can you get #dep to add it?

Comment (zaipie) [reply content/en/docs/appstore/modules/mendix-sso.md]: Hi @MarkvanMents
I haved asked #dep to add this, it is now a regular supported environment variable. It is available on production.

File: content/en/docs/appstore/modules/mendix-sso.md [hunk 2]
```diff
@@ -274,6 +274,45 @@ Remember that data which comes from the end-user's Mendix ID via SSO (for exampl
 
 The [Administration](https://marketplace.mendix.com/link/component/23513) module versions 1.3.X (for example 1.3.2) and 2.1.X (for example 2.1.2) contain a set of microflows to configure Mendix SSO to use **Administration.Account** as the user entity. Follow the instructions in [Using the Administration Module with Mendix SSO](/appstore/modules/administration/#use-with-mendix-sso) to use the Administration module with Mendix SSO.
 
+### 5.4 Using the Continuation URL Parameter
+
+{{% alert color="info" %}}
+Continuation URLs were introduced in version 4.2.0 of the Mendix SSO module.
```
Comment (zaipie) [inline content/en/docs/appstore/modules/mendix-sso.md]: Continuation URLs have been part of MendixSSO for a long time. I would prefer to move this to section 5.4.1 and rewrite is as:
Continuation URL validation was introduced in version 4.2.0 of the Mendix SSO module.

File: content/en/docs/appstore/modules/mendix-sso.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/mendix-sso.md]: Changed https://github.com/mendix/docs/pull/6507/commits/8baae8624a45ecc17b15f9bfce0dee2f3e48d3b4

File: content/en/docs/appstore/modules/mendix-sso.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/mendix-sso.md]: Changed https://github.com/mendix/docs/pull/6507/commits/8baae8624a45ecc17b15f9bfce0dee2f3e48d3b4

**Comment by MarkvanMents** [comment]
@zaipie 
OK to publish now?

**Comment by MarkvanMents** [comment]
Changes approved over Slack.
Waiting for module version 4.2.0 to be released.

**Comment by MarkvanMents** [comment]
Release approved over Slack.

**Pull request merged**